openhtf/conf.py

```diff
@@ -257,29 +256,31 @@ def Load(self, config_file=None, force_reload=False,
       return False
 
     try:
-      filename = config_file or FLAGS.config
-      _LOG.info('Loading from config: %s', filename)
-
-      with config_loader(filename) as config_file:
-        data = yaml.safe_load(config_file)
-        if not data:
-          raise ConfigurationInvalidError('No data', config_file)
-        self._state.clear()
-        self._state.update(data)
+      if FLAGS.config:
+        _LOG.info('Loading from config: %s', FLAGS.config)
+        with config_loader(FLAGS.config) as config_file:
+          data = yaml.safe_load(config_file)
+          if not data:
+            raise ConfigurationInvalidError('No data', config_file)
+          self._state.clear()
+          self._state.update(data)
 
       # Load string values from flags
       for keyval in FLAGS.config_value:
         key, val = keyval.split('=')
         self._state[key] = val
+   
+      if 'station_id' not in self._state:
```

Comment:
The rundata module uses it as we see, and the built-in JSON output module uses it as well.